impl/src/main/java/org/jboss/forge/furnace/container/guice/lifecycle/GuiceAddonLifecycleProvider.java

```diff
@@ -54,12 +57,14 @@ public void start(Addon addon) throws Exception
    {
       Set<Module> modules = new LinkedHashSet<>();
       modules.add(new GuiceContainerModule(furnace, addonRegistry, addon));
+      modules.add(binder -> binder.bindListener(Matchers.any(), new ServiceTypeListener(addonRegistry)));
+
       for (Module addonModule : ServiceLoader.load(Module.class, addon.getClassLoader()))
       {
          modules.add(addonModule);
       }
 
-      this.injector = Guice.createInjector(modules);
+      this.injector = Guice.createInjector(Stage.PRODUCTION, modules);
```

Comment:
See https://google.github.io/guice/api-docs/latest/javadoc/index.html?com/google/inject/Stage.html

Since we create an injector for user, we should set production stage for the safety.